server/controllers/comment.js

```diff
@@ -0,0 +1,235 @@
+import models from '@models';
+import validateComment from '@validations/comment';
+import { validationResponse } from '@helpers/validationResponse';
+import Response from '@helpers/Response';
+
+const { Comment, Article } = models;
+
+/**
+ * @exports CommentController
+ * @class CommentController
+ * @description Handles the comment section
+ */
+class CommentController {
+  /**
+   * Create new comment
+   * @static
+   * @param {object} req - The Request Object
+   * @param {object} res - The Response Object
+   * @param {object} next - The Next Middleware
+   * @return {json} - Returns json Object
+   * @memberof CommentController
+   * @static
+   */
+  static async create(req, res, next) {
+    try {
+      const data = await validateComment(req.body);
+      const body = data.comment;
+      const { slug } = req.params;
+      const userId = req.decoded.id;
+
+      const article = await Article.findOne({ where: { slug } });
+
+      if (!article) return Response.error(res, 404, 'Article does not exist');
+
+      const comment = await article.createComment({ userId, slug, body });
+
+      const payload = await Comment.findAll({
```

Comment:
abstract this thing o baba.

You would find that you are doing the same thing in multiple places.

Don't Repeat Yourself (DRY)

Reply:
on it now

Reply:
Feedback implemented